src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+
+            DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
+
+            Assert.NotNull(directorySecurity);
```

Comment:
public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo)
{
            return new DirectorySecurity(directoryInfo.FullName, AccessControlSections.Access | AccessControlSections.Owner | AccessControlSections.Group);
}

@stephentoub @ianhays any suggestion on how to get AccessControlSections.Access,  AccessControlSections.Owner and AccessControlSections.Group out of the DirectorySecurity object? Thanks, Michael